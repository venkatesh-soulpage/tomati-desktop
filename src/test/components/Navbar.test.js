import React from "react";
import { render } from "@testing-library/react";
import Navigation from "components/Navbar";
import { Router } from "react-router-dom";
import history from "utils/history";
import { Provider } from "react-redux";
import configureStore from "store";

describe("Navigation page", () => {
  it("matches snapshot", () => {
    const store = configureStore();
    const { asFragment } = render(
      <Provider store={store}>
        <Router history={history}>
          <Navigation />
        </Router>
      </Provider>,
      {}
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
