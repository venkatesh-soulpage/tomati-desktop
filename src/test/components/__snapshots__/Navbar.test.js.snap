// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="fixed-top bg-white border-bottom"
    style="z-index: 9999999;"
  >
    <div
      class="container"
    >
      <nav
        class="navbar navbar-expand-md navbar-light"
      >
        <a
          class="navbar-brand"
          href="/"
        >
          <img
            alt="Tomati.app"
            class="img-fluid d-inline-block align-top"
            height="50"
            src="Logo.svg"
            width="150"
          />
        </a>
        <button
          aria-controls="basic-navbar-nav"
          aria-label="Toggle navigation"
          class="border-0 navbar-toggler collapsed"
          type="button"
        >
          <img
            alt="toggle"
            height="50"
            src="toggle.svg"
            width="50"
          />
        </button>
        <div
          class="navbar-collapse collapse"
          id="basic-navbar-nav"
        >
          <div
            class="ml-auto navbar-nav"
            style="font-size: smaller;"
          />
        </div>
      </nav>
    </div>
  </div>
</DocumentFragment>
`;
